actions/build-lzo

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+import os
+import glob
+import shutil
+import hashlib
+from git import Repo, Git
+from charmhelpers.core import hookenv
+from subprocess import check_call
+import json
+import buildutils
+from shutil import copyfile
+
+
+LZO_GIT = 'https://github.com/twitter/hadoop-lzo.git'
+LZO_VERSION = hookenv.action_get("lzo-version")
+
+
+def get_prerequisits(builder):
+    hookenv.log("Fetching required packages")
+    package_names = ['git', 'default-jre', 'openjdk-7-jdk', 'liblzo2-2', 'liblzo2-dev', 'maven']
+    builder.install_apt_packages(package_names)
+
+
+def checkout_source(builder):
+    '''
+    Checkout source and switch to the right tag
+    '''
+    hookenv.log("Fetching source from {}".format(LZO_GIT))
+    builder.checkout_source(LZO_GIT)
+
+
+def build_binary(builder):
+    hookenv.log("Building lzo libraries")
+    build_command = ['mvn', 'clean', 'package']
+    builder.build_binary(build_command)
+    os.makedirs(builder.workspace + '/share/hadoop/common/lib')
+    os.makedirs(builder.workspace + '/lib/native')
+    copyfile(builder.workspace + '/target/hadoop-lzo-' + LZO_VERSION + '.jar', builder.workspace + '/share/hadoop/common/lib/hadoop-lzo-' + LZO_VERSION + '.jar')
```

Comment:
The output directory has a "lzo" (builder.workspace + '/lzo/target/hadoop-lzo-'........ )

2016-05-12 09:12:55 INFO build-lzo Traceback (most recent call last):
2016-05-12 09:12:55 INFO build-lzo   File "/var/lib/juju/agents/unit-gobblin-binary-builder-0/charm/actions/build-lzo", line 84, in <module>
2016-05-12 09:12:55 INFO build-lzo     build_binary(builder)
2016-05-12 09:12:55 INFO build-lzo   File "/var/lib/juju/agents/unit-gobblin-binary-builder-0/charm/actions/build-lzo", line 39, in build_binary
2016-05-12 09:12:55 INFO build-lzo     copyfile(builder.workspace + '/target/hadoop-lzo-' + LZO_VERSION + '.jar', builder.workspace + '/share/hadoop/common/lib/hadoop-lzo-' + LZO_VERSION + '.jar')
2016-05-12 09:12:55 INFO build-lzo   File "/usr/lib/python3.4/shutil.py", line 108, in copyfile
2016-05-12 09:12:55 INFO build-lzo     with open(src, 'rb') as fsrc:
2016-05-12 09:12:55 INFO build-lzo FileNotFoundError: [Errno 2] No such file or directory: '/tmp/workspace/target/hadoop-lzo-0.4.20-SNAPSHOT.jar'
^